pkg/api/index.go

```diff
@@ -358,6 +358,22 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
 			SortWeight:   dtos.WeightAdmin,
 			Children:     adminNavLinks,
 		})
+
+		saasNavLinks := []*dtos.NavLink{
+			{Text: "Cluster", Id: "dbaas-cluster", Url: "/graph/d/pmm-dbaas/cluster", Icon: "database"},
+			{Text: "Inventory", Id: "dbaas-iventory", Url: "/graph/d/pmm-dbaas/inventory", Icon: "building"},
```

Comment:
We only need one for now since the "Manage DB Cluster" won't exist yet.

Reply:
Removed that as discussed.